lib/Doctrine/Common/EventManager.php

```diff
@@ -68,7 +70,7 @@ public function dispatchEvent($eventName, EventArgs $eventArgs = null)
      *
      * @return array The event listeners for the specified event, or all event listeners.
      */
-    public function getListeners($event = null)
+    public function getListeners(?string $event = null): array
```

Comment:
The fact that `string $event = null` works is sort of a flaw of PHP 7.0 where nullable types didn't exist and nullability was implied by default null value.
Anyhow, when I talk about this difference with other people, I usually use this example to show why explicit `?` matters (besides consistence):

```php
function foo(?int $bar = null) {} // accepts int or null
function bar(?int $arg = 123) {} // accepts int or null
```
vs.
```php
function foo(int $arg = null) {} // accepts int or null
function bar(int $arg = 123) {} // oops, accepts only int, no null here!
```
This is even more obvious in inheritance:
```php
class A {
    public function foo(int $arg = null) {}
}
class B extends A {
    public function foo(int $arg = 123) {} // this is an error, LSP violation
}
// PHP Warning:  Declaration of B::foo(int $arg = 123) should be compatible with A::foo(?int $arg = NULL)
```